internal/engine/experiment/tor/tor.go

```diff
@@ -178,10 +176,6 @@ func (m *Measurer) Run(
 	if err != nil {
 		return err
 	}
-	ctx, cancel := context.WithTimeout(
-		ctx, 15*time.Second*time.Duration(len(targets)),
-	)
-	defer cancel()
```

Comment:
We don't need an overall timeout for the whole measurement here because each "easy" API call has its own internal timeout, hence it's more effective to let each timeout do its job. The measurement could run for longer but it could also be more precise because it could measure _more_ targets.